operator/overlays/default-namespace/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+bases:
```

Comment:
`bases` field is [deprecated](https://kubernetes-sigs.github.io/kustomize/blog/2019/06/18/v2.1.0/#field-changes--deprecations).

Reply:
thanks, fixed 👍  (didn't knew 😇  )